boilerplate/flyte/golang_support_tools/go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/flyteorg/boilerplate
 
-go 1.17
+go 1.16
 
 require (
 	github.com/alvaroloes/enumer v1.1.2
-	github.com/flyteorg/flytestdlib v0.3.22
+	github.com/flyteorg/flytestdlib v0.4.7
 	github.com/golangci/golangci-lint v1.38.0
-	github.com/pseudomuto/protoc-gen-doc v1.4.1 // indirect
 	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5
+	github.com/pseudomuto/protoc-gen-doc v0.1.1 // indirect
```

Comment:
Why is gen-doc also going to a lower version

Reply:
It doesn't matter because in the we are using our fork in replace

Reply:
But can we understand what caused this downgrade even though we have a replace.

Reply:
go mod is considering it because it doesn't use anywhere in code, So go.sum doesn't need it in require because we are installing it from fork repo in replace.

protoc-gen-doc's fork has an entry in go.sum that's it is working

Reply:
fixed the issue https://github.com/flyteorg/boilerplate/pull/49